### DISCUSSION TITLE
Won't insert an include for unsolved symbols in std::

### DISCUSSION THREAD

**Discussion by wenhao-huu** [clangd/clangd#1562] [Q&A] [answered]
As metioned in #618 , but when I type 'std::', nothing is suggested, unless some headers have been included.
![image](https://user-images.githubusercontent.com/36889966/229006543-a860674f-dd68-4ea2-b2a3-34689a674835.png)

Clangd arguments are as follow:
![image](https://user-images.githubusercontent.com/36889966/229007692-6841c464-1fd4-484e-969f-3a8de06633a3.png)

verbose log file:  
[clangd.verbose-log.txt](https://github.com/clangd/clangd/files/11117614/clangd.verbose-log.txt)

**Comment by HighCommander4** [comment, marked as answer]
This can be enabled by adding the following to your `.clangd` file (https://clangd.llvm.org/config.html#standardlibrary):

```yaml
Index:
  StandardLibrary: Yes
```

In clangd 16 this will be the default.

**Reply by wenhao-huu** [reply]
THANK YOU SO MUCH!!
